tests/unit/utils/sparqlhandler/test_csvw_inspector.py

```diff
@@ -55,5 +56,48 @@ def test_detect_csvw_type_code_list():
     )
     csvw_rdf_manager = CsvwRdfManager(csvw_metadata_json_path)
 
-    csvw_type = csvw_rdf_manager.csvw_state.csvw_type
+    csvw_type = csvw_rdf_manager.csvw_inspector.csvw_type
     assert csvw_type == CSVWType.CodeList
+
+
+def test_get_table_schema_properties():
+    """
+    Ensures that the correct table schema properties are returned for the given code list.
+    """
+    csvw_metadata_json_path = (
+        _test_case_base_dir
+        / "pivoted-single-measure-dataset"
+        / "some-dimension.csv-metadata.json"
+    )
+    csvw_rdf_manager = CsvwRdfManager(csvw_metadata_json_path)
+
+    csvw_inspector: CsvWInspector = CsvWInspector(
```

Comment:
We should probably agree on some consistent case sensitivity of `CsvW` or `Csvw` in class names to make it more predictable. Any thoughts?

Reply:
Personal preference for CsvW

Reply:
We should probably rename the `CsvwRdfManager` to `CsvWRdfManager` then.

Reply:
Done